packages/annotations/src/search.ts

```diff
@@ -105,3 +108,167 @@ export function searchAnnotations(
+    upVotes: number;
+    downVotes: number;
+  };
+}
+
+/**
+ * ```js
+ * import { searchSingleAnnotationVotes } from "@esri/hub-annotations";
+ * //
+ * searchSingleAnnotationVotes({ url: annotationsUrl + "/0",
+ *                         annotation: {
+ *                            id: "Annotation1",
+ *                            type: "annotations",
+ *                            attributes: {description: "Great place!", ...}
+ *                          })
+ *   .then(response => {
+ *     //   data: [{
+ *     //     id,
+ *     //     type: "votes",
+ *     //     attributes: {
+ *     //       upVotes: 3,
+ *     //       downVotes: 0
+ *     //     }
+ *     //   }]
+ *    });
+ * ```
+ * Query for up and down votes on a comment from ArcGIS Hub.
+ * @param requestOptions - request options that may include authentication
+ * @param annotation - the annotation for which votes need to be counted
+ * @returns A Promise that will resolve with summary statistics for the specified annotation from the annotation service for a Hub enabled ArcGIS Online organization.
+ */
+export function searchSingleAnnotationVotes(
+  requestOptions: ISearchAnnoRequestOptions
+): Promise<{ data: IVoteResourceObject[] }> {
+  const data: IVoteResourceObject[] = [];
+  const annotationId = requestOptions.annotation.attributes.OBJECTID;
+  if (!annotationId || annotationId < 0) {
+    return Promise.resolve({ data });
+  }
+  requestOptions.groupByFieldsForStatistics = "value";
+  const outStat: IStatisticDefinition = {
+    statisticType: "count",
+    onStatisticField: "value",
+    outStatisticFieldName: "value_count"
+  };
+  requestOptions.outStatistics = [outStat];
+  // filtering for the comment
+  requestOptions.where = "parent_id=" + annotationId;
+  const queryRequestOptions = requestOptions as IQueryFeaturesRequestOptions;
+  return queryFeatures(queryRequestOptions).then(response => {
+    const resource: IVoteResourceObject = {
+      id: annotationId,
+      type: "votes",
+      attributes: {
+        upVotes: 0,
+        downVotes: 0
+      }
+    };
+
+    // use .reduce()?
+    (response as IQueryFeaturesResponse).features.forEach(
+      (statistic: IFeature) => {
+        if (statistic.attributes.value > 0) {
+          resource.attributes.upVotes += statistic.attributes.value_count;
+        } else if (statistic.attributes.value < 0) {
+          resource.attributes.downVotes += statistic.attributes.value_count;
+        }
+      }
+    );
+    data.push(resource);
+    return { data };
+  });
+}
+
+/**
+ * ```js
+ * import { searchAnnotationVotes } from "@esri/hub-annotations";
+ * //
+ * searchAllAnnotationVotes({ url: annotationsUrl + "/0"})
+ *   .then(response => {
+ *     //   data: [{
+ *     //     id,
+ *     //     type: "votes",
+ *     //     attributes: {
+ *     //       upVotes: 3,
+ *     //       downVotes: 0
+ *     //     }
+ *     //   }]
+ *    });
+ * ```
+ * Query for up and down votes on a comment from ArcGIS Hub.
```

Comment:
```
Query for up and down votes on multiple comments from ArcGIS Hub.
```

Reply:
it looks like you mixed up the descriptions for your two new methods, when you made this update.